vendor/bat-native-confirmations/src/bat/confirmations/internal/confirmations_client_mock.h

```diff
@@ -136,7 +136,9 @@ class MockConfirmationsClient : public ConfirmationsClient {
       ledger::Result result,
       const ledger::Grant& grant));
 
-  MOCK_METHOD0(GetGrantCaptcha, void());
+  MOCK_METHOD0(GetGrantCaptcha, void(
```

Comment:
This should be MOCK_METHOD2 otherwise the build would still fail.

Reply:
@yrliou woops, fixed :)